### ISSUE TITLE
Allow passing options to action

### ISSUE THREAD

**Issue by svrcekmichal** [svrcekmichal/redux-axios-middleware#15]
Idea is to allow passing options like `returnRejectedPromiseOnError ` to action.  

```javascript
{
  type: 'LOAD',
  payload: {
    request:{
      url:'/categories',
      options: {
        returnRejectedPromiseOnError: true
      }
    }
  }
}
```

This way user can override client options in middleware setup. Options will be merged:

default options => middleware options => action options

**Comment by svrcekmichal** [comment]
#17 With support of multiple clients we now have new structure:
default options => middleware options => client options => action options
You can't do all changes on all layers, but you can find what you can do in documentation

**Comment by svrcekmichal** [comment]
closed with #17
